app/services/AttachmentsService.scala

```diff
@@ -58,7 +58,9 @@ class AttachmentsService @Inject()(val registrationRepository: VatSchemeReposito
       getVat2Attachment(vatScheme),
       getVat51Attachment(vatScheme),
       getTaxRepresentativeAttachment(vatScheme),
-      getVat5LAttachment(vatScheme)
+      getVat5LAttachment(vatScheme),
+      get1614aAttachment(vatScheme),
```

Comment:
This list is only set up for mandatory attachments with how it interacts with the FE, adding optional ones to it breaks the FE attachments journey. If you rebase there's now an optionalAttachmentList you could use instead.